src/components/Star.js

```diff
@@ -3,21 +3,21 @@ import StarDefault from '../assets/starDefault.svg';
 import StarSeleted from '../assets/starSeleted.svg';
 
 function Star() {
-    const [starState, setStarState] = useState('default');
+    const [isStarShowing, setIsStarShowing] = useState(false);
 
-    const handleStarSeleted = (e) => {
-        if (starState === 'default') {
-            setStarState('seleted');
+    const handleStarSeleted = () => {
+        if (isStarShowing === false) {
+            setIsStarShowing(true);
         } else {
-            setStarState('default');
+            setIsStarShowing(false);
```

Comment:
이전값을 그대로 활용할 수도 있어요! 
```suggestion
        setIsStarShowing(!isStarShowing)
```